tests/pom.xml

```diff
@@ -164,6 +164,149 @@
                 </dependency>
             </dependencies>
         </profile>
+        <profile>
+            <id>release311db</id>
+            <activation>
+                <property>
+                    <name>buildver</name>
+                    <value>311db</value>
+                </property>
+            </activation>
+            <dependencies>
+                <dependency>
+                    <groupId>org.apache.spark</groupId>
+                    <artifactId>spark-sql_${scala.binary.version}</artifactId>
+                    <version>${spark311db.version}</version>
+                    <scope>provided</scope>
+                </dependency>
+                <dependency>
+                    <groupId>org.apache.spark</groupId>
+                    <artifactId>spark-core_${scala.binary.version}</artifactId>
+                    <version>${spark311db.version}</version>
+                    <scope>provided</scope>
+                </dependency>
+                <dependency>
+                    <groupId>org.apache.spark</groupId>
+                    <artifactId>spark-network-common_${scala.binary.version}</artifactId>
```

Comment:
This seems like a second-level dependency which we shouldn't have to list here.  If we do depend on it directly, why isn't this or others listed for the other profiles?

Reply:
copy and paste error. I cleaned up, but I had lost a bunch of changes due to data bricks instance shutdown on me, thought I remade but missed some. There are a few others that go along with this.